README.md

```diff
@@ -46,7 +46,7 @@ try {
 // You can also initiate XenCrypt using file path name
 
     val xenditPublicKeyPathFileName = "src/path_to_file/publickey.crt"
-    val xenIssuing = XenIssuing(null, xenditPublicKeyPathFileName)
+    val session = SecureSession(null, xenditPublicKeyPathFileName)
```

Comment:
I don't think we need to support both ways of passing in the public key. String should be good enough. 

Side note @sami is there way to get the key from the internet instead? In case we need to change it?

Reply:
i think s3 was a good idea